src/dyncfg/src/lib.rs

```diff
@@ -563,8 +612,10 @@ mod tests {
     const BOOL: Config<bool> = Config::new("bool", true, "");
     const USIZE: Config<usize> = Config::new("usize", 1, "");
     const OPT_USIZE: Config<Option<usize>> = Config::new("opt_usize", Some(2), "");
-    const STRING: Config<String> = Config::new("string", "a", "");
+    const STRING: Config<&'static str> = Config::new("string", "a", "");
```

Comment:
nit: Do we need the `'static`? Seems like the one on compute is fine with just `&str`

Reply:
Ah yes, for sure!

Reply:
Fixed.